README.md

```diff
@@ -39,73 +41,68 @@ source env.sh
 
+`candigv2-ingest` can be used as either a command-line tool, a local API server or a docker container. To run the command line scripts, set your environment variables and follow the command line instructions in the sections below. To use the local API, set your environment variables, run `python app.py`, and follow the API instructions in the sections below. The API will be available at `localhost:1236`. A swagger UI is also available at `/ui`. Docker instructions can be found at the [bottom of this document](#Run-as-Docker-Container). To authorize yourself for these endpoints, you will need to set the Authorization header to a keycloak bearer token (in the format `"Bearer ..."` without the quotes).
 
-## Authorizing users for the new dataset
-### NOTE: OPA ingest is currently not functional, so these instructions will not work.
-Create a new access.json file:
+## 1. Clinical data
 
-```bash
-python opa_ingest.py --dataset <dataset> --userfile <user file> > access.json
-```
+### i. Prepare clinical data
 
-Alternately, you can add a single user:
+Before being ingested, data must be transformed to the CanDIG MoH data model. This can be done using CanDIG's [`clinical_ETL_code`](https://github.com/CanDIG/clinical_ETL_code) repository. Please visit that repository for full instructions and return to ingest when you have a valid JSON file with a set of donors. An example file can be found at [tests/single_ingest.json](tests/single_ingest.json)
 
-```bash
-python opa_ingest.py --dataset <dataset> --user <user email> > access.json
-```
+### ii. Ingest clinical data
 
-If you're running OPA in the CanDIGv2 Docker stack, you should copy the file to the Docker volume to persist the change between restarts:
+The preferred method for clinical data ingest is using the API.
 
-```bash
-docker cp access.json candigv2_opa_1:/app/permissions_engine/access.json
-```
+#### API
 
-Restart the OPA container to take effect
+The clinical ingest API runs at `/ingest/clinical`. Simply send a request with an authorized bearer token and a JSON body with your `DonorWithClinicalData` object. See the swagger UI/[schema](ingest_openapi.yaml) for the response format.
 
-## Ingest clinical data
-Note: To upload a DonorWithClinicalData object, you should produce one using the [CanDIG ETL](https://github.com/CanDIG/clinical_ETL_code), then store its result in a JSON file under the key "donors". See single_ingest.json for an example.
+#### Command line
 
-### Command line
-Before you can ingest the clinical data, you need to format your data into the json ingest format using the [clinical_ETL](https://github.com/CanDIG/clinical_ETL_data) and put it in the katsu `data` folder, then set the environment variable `CLINICAL_DATA_LOCATION`:
+This method is mainly used for development work. 
```

Comment:
For now, this is also used for ingest of jsons that are too big to send easily via POST.